assets/css/admin/style.scss

```diff
@@ -2414,6 +2414,45 @@ span.edd-to-and-from--separator {
 	font-size: 1.5em;
 }
 
+.edd-reports-tile .tile-relative {
+	text-align: center;
+	font-size: 11px;
+	font-weight: normal;
+	color: #888;
+	order: 2;
+	margin-top: 3px;
+	display: block;
+}
+
+.edd-reports-tile .tile-relative span.dashicons {
+	display: inline-block;
+	font-size: 22px;
+	line-height: 20px;
+	height: 20px;
+	width: 20px;
+	position: relative;
+	top: -4px;
+	left: 3px;
+	margin-left: 0px;
+	color: #999;
+}
+
+.edd-reports-tile .tile-relative span.dashicons-arrow-down {
+	color: #FF6666;
+}
+
+.edd-reports-tile .tile-relative span.dashicons-arrow-up {
```

Comment:
Minor thing but since we have four rules which are really two pairs of rules could they be combined/reduced to two? Also would love if we used existing WP Core colors or defined/used a red/green ourselves. We have a `$wp-red-50` and a `$wp-green-50` already defined, for example.